src/server/ws-routes/initializeWebSocketRouting.js

```diff
@@ -0,0 +1,10 @@
+const ConnectionsRepository = require('./ConnectionsRepository')
+
+const Utilities = require('../utilities')
+
+
+module.exports = initializeWebSocketRouting
```

Comment:
move `module.exports = initializeWebSocketRouting` to end line of file.

Reply:
oka